tests/storage/conftest.py

```diff
@@ -115,18 +117,36 @@ def hpp_resources(request, admin_client):
 @pytest.fixture(scope="module")
 def internal_http_configmap(namespace, internal_http_service, workers_utility_pods, worker_node1):
     svc_ip = internal_http_service.instance.to_dict()["spec"]["clusterIP"]
-    cert = ExecCommandOnPod(utility_pods=workers_utility_pods, node=worker_node1).exec(
-        command=(
-            f"openssl s_client -showcerts -connect {svc_ip}:443 </dev/null 2>/dev/null|"
-            "sed -n '/-----BEGIN/,/-----END/p'"
-        )
-    )
-    with ConfigMap(
-        name=INTERNAL_HTTP_CONFIGMAP_NAME,
-        namespace=namespace.name,
-        data={"tlsregistry.crt": cert},
-    ) as configmap:
-        yield configmap
+
+    def _fetch_cert():
+        try:
+            return ExecCommandOnPod(utility_pods=workers_utility_pods, node=worker_node1).exec(
+                command=(
+                    f"openssl s_client -showcerts -connect {svc_ip}:443 </dev/null 2>/dev/null | "
+                    "sed -n '/-----BEGIN/,/-----END/p'"
+                )
+            )
+        except workers_utility_pods.ExecOnPodError:
```

Comment:
If I understand it correctly,  `ExecOnPodError` is not an attribute of `pod` instances. Should it be

```
from ocp_resources.exceptions import ExecOnPodError
  ...
  except ExecOnPodError:
```